server/models/profile.js

```diff
@@ -0,0 +1,55 @@
+const mongoose = require("mongoose");
+const avaliableTime = new mongoose.Schema({
```

Comment:
Please adjust the base of the PR/rebase to only reflect changes pertaining to this PR

Reply:
Done the changes.

Reply:
The changes are still being reflected within this PR :)